guides/plugin-advisories.md

```diff
@@ -0,0 +1,203 @@
+
+- [ ] On the (privately) published page, under "Text version for reports" click the "Click for a blank email with the right subject" link
+- [ ] Copy the email text from "Text version for reports"
+- [ ] Find contact details for the plugin author
+- [ ] Add a humanising intro and send to the plugin author
+- [ ] Add a note to the timeline, mentioning the method - e.g. 2014-01-02: Reported to vendor by email
+- [ ] At the bottom of the page, set the status to "Reported"
+
+As an example of a humanising intro:
+
+> Hello
+> We’ve found a security vulnerability in Version 2.8.2 of the Advanced Access Manager plugin – please see the details below.
+> After you’ve had a chance to investigate could you please let us know a timeline for a fix – our disclosure policy is linked at the bottom of the report.
+> Best regards
+> Duncan
+
+### Request a CVE
+
+- [ ] Hit the "Request CVE" button under "Report emails"
+- [ ] Add a note to the timeline - e.g. 2014-01-02: Requested CVE
+
+### Notify the WordPress plugin admins
+
+There are two scenarios in which the WordPress plugin admins (plugins@wordpress.org) should be contacted:
+
+1. If we really couldn’t get in touch with the plugin dev
+2. If the vulnerability is really serious and the plugin is popular (say over 100,000 downloads), so needs to be taken down ASAP
+
+Their policy is this:
+
+> Every attempt to contact the developer directly should be made BEFORE you reported the plugin to us. Since our policy is to close the plugin (preventing new downloads) until it’s fixed, you may not be alerted until the plugin is updated.
+
+- [ ] Hit the "WP Plugins" button under "Report emails"
+- [ ] Add a note to the timeline - e.g. 2014-01-02: Reported to WP Plugins
+
+#### :rotating_light: IMPORTANT :rotating_light:
+
+If we couldn’t get in touch with the plugin author then WP Org may be able to contact them on our behalf, BUT we shouldn’t expect them to feed back any information to us, or even to confirm that they have successfully contacted the author (they are insanely busy).
+
+Therefore we shouldn’t say things like “No response from WP Org” (because we shouldn’t expect one) or “No response from author to WP Org” (because we don’t know that’s the case).
+
+## Publishing an Advisory
+
+There are two reasons that we will publish an advisory:
+
+- The plugin author lets us know that they’ve fixed the issue and released a fix
+- We couldn’t contact the plugin author, or they didn’t respond within the
+  time period defined in our [disclosure
+  policy](https://security.dxw.com/disclosure/)
+
+If an author has at any point been in touch and shown willingness to cooperate
+then we should give them the benefit of the doubt and make several attempts to
+get in touch.
+
+### Preparing advisories for publication
+
+If the issue is fixed:
+
+- [ ] Email the developer to thank them for the fix and let them know we'll be publishing.
```

Comment:
I never follow this step. You can send your thanks when you send them the link. No point sending two emails when you can send one.